zyanken.py

```diff
@@ -0,0 +1,14 @@
+print('{:-^4}'.format('-じゃんけんゲームが始まるぞ！！-\n'))
+print('{:^30}'.format('入力した数だけじゃんけんできるぞ！！\n↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓\n'))
+
+while True:
+    try:
+        n = int(input('じゃんけんしたい回数を入力してね\n--->'))
```

Comment:
type annotationを付けましょう。
```suggestion
        n: int = int(input('じゃんけんしたい回数を入力してね\n--->'))
```

また、integer以外は排除できますが、0以下を入力した場合はどうしますか?

Reply:
Revをしていただきありがとうございます。
type annotationを付けました。
０以下の入力をした際に、エラーを起こしてValue Errorの処理を行うように変更しました。